f2/src/forms/LocationInfoForm.js

```diff
@@ -55,24 +56,23 @@ export const LocationInfoForm = (props) => {
               />
             )}
             <Container>
-              <Row>
+              <FormRow>
                 <Trans id="locationinfoPage.skipInfo" />
-              </Row>
-              <Row>
+              </FormRow>
+              <FormRow marginBottom="1rem">
                 <SkipButton
                   label={<Trans id="locationinfoPage.skipButton" />}
                   to="/contactinfo"
                 />
-              </Row>
-              <br />
-              <Row>
+              </FormRow>
+              <FormRow>
                 {errors.postalCode && (
                   <P color="#dc3545" fontSize="1.25rem" marginBottom="0.5rem">
                     {errors.postalCode}
                   </P>
                 )}
-              </Row>
-              <Row className="form-section">
+              </FormRow>
+              <FormRow className="form-section">
```

Comment:
Please remove the className property here. I think it would be a good idea to move away from this now that we are refactoring the components to use  a CSS-in-JS approach.  If there are any style properties this FormRow needs you can pass them directly now.

Reply:
advice taken

Reply:
All updated in latest push